cms/static/js/views/course_info_update.js

```diff
@@ -17,32 +23,53 @@ define(['js/views/validation', 'codemirror', 'js/models/course_update',
 
             initialize: function() {
                 this.template = this.loadTemplate('course_info_update');
-                this.render();
```

Comment:
So no call to render in initialize because the collection listeners will cause it to be called later?